play.py

```diff
@@ -0,0 +1,89 @@
+class Game:
+    def __init__(self, board, ui):
+        self.ui = ui
+        self.board = board
+
+    def play(self):
+        self.ui.display("Welcome to Battleship")
+        self.ui.display("Take your best shot")
+        self.ui.display(self.board.format())
+        valid_spot = ValueError
+        while valid_spot == ValueError:
+            valid_spot =  self.board.validate(self.ui.get_input('>>'))
+        self.board.update(valid_spot)
+        self.ui.display(self.board.format())
+
+class TerminalUi:
+    def display(self, message):
+       print(message)
+
+    def get_input(self, prompt_string):
+        response = input(prompt_string)
+        return response
+
+class Board:
+    def __init__(self):
+        self.state = [[None for ele in range(10)] for index in range(10)]
+
+    def format(self):
+        formatted_board = ''
+        for row in range(0, len(self.state)):
+            formatted_board += '\n'
+            for column in range(0, len(self.state[row])):
+                if self.state[column][row] is None:
+                    formatted_board += '[ ]'
+                else: 
+                    formatted_board += '[X]'
+        formatted_board += '\n'
+        return formatted_board
+    
+    def validate(self, user_shot_choice):
+        letters = [chr(i) for i in range(ord('A'), ord('J')+1)]
```

Comment:
Cool use of range and ord! But you calculate this here and it is a constant in another method of the same object